src/components/profile/MemberProjectsContainer.js

```diff
@@ -2,8 +2,22 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 
 import ProjectCard from '../cards/ProjectCard';
+import AddProject from '../AddProject';
+import AddButton from '../buttons/AddButton';
 
 class MemberProjectsContainer extends Component {
+  state = {
+    showAddProjectForm: false
+  };
+
+  handleAddButtonClick = () => {
+    this.setState({ showAddProjectForm: true });
```

Comment:
Add a placeholder method here and update `MockApiService.js` and `ApiService.js` 

Pass the project created based on the form data `handleAddButtonClick(project)` and then update handleAddButtonClick with call to createProject endpoint. 

```suggestion
    await api.createProject(project);
    this.setState({ showAddProjectForm: true });
```